storybook/stories/mural-account-chooser.stories.tsx

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
-import { ACCOUNT_CHOOSER_ACTION } from '../packages/mural-account-chooser/dist';
-import AccountChooser, {
-  AuthorizeParams,
-} from '../packages/mural-account-chooser/src/components/account-chooser';
+import {
+  AccountChooser,
+  ACCOUNT_CHOOSER_ACTION,
+} from '@muraldevkit/mural-integrations-mural-account-chooser';
+
+// TODO: Remove this
```

Comment:
👀

Reply:
Did you mean to say "Remove this before merging the PR" or "Remove this eventually...."?